pkg/minikube/cluster/cluster.go

```diff
@@ -94,6 +94,9 @@ func StartHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 		if err := api.Save(h); err != nil {
 			return nil, fmt.Errorf("Error saving started host: %s", err)
 		}
+		if err := setProxyToShell(config, h); err != nil {
+			return nil, err
```

Comment:
@praveenkumar May be should add a meaningful string to the err e.g. `fmt.Errorf("Error setting proxy variables to shell : %s", err)